tmt/utils.py

```diff
@@ -735,6 +736,24 @@ def context_to_dict(context):
         for key, value in environment_to_dict(context).items()}
 
 
+def dict_to_shell(data, width=None, sort=False):
+    if not data:
+        return "\n"
+    output = ""
+    # List available attributes
+    for key, value in sorted(data.items()):
+        output += "{0}: ".format(fmf.utils.color(key, 'green'))
+        if isinstance(value, type("")):
+            output += value.rstrip("\n")
+        elif isinstance(value, list) and all(
+                [isinstance(item, type("")) for item in value]):
+            output += fmf.utils.listed(value)
+        else:
+            output += pretty(value)
```

Comment:
We should find a nice representation for a dict as it can be present even in require/recommend (hint: library)

Reply:
Example of a test using require with dict (library)
```
recommend: mariadb and mariadb-server
require: [{'name': '/mariadb',
  'ref': 'master',
  'url': 'https://github.com/beakerlib/database'},
 'expect',
 'beakerlib']
result: respect
```